torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -653,26 +653,25 @@ void RRefContext::recordThreadLocalPendingRRefs() {
   recording_ = true;
 }
 
-std::shared_ptr<Future<bool>> RRefContext::waitForThreadLocalPendingRRefs() {
-  std::shared_ptr<Future<bool>> future;
+c10::intrusive_ptr<JitFuture> RRefContext::waitForThreadLocalPendingRRefs() {
+  auto jitFuturePtr = c10::make_intrusive<JitFuture>(BoolType::get());
   if (userTable_.empty()) {
-    future = std::make_shared<Future<bool>>(true);
+    jitFuturePtr->markCompleted();
```

Comment:
would I be correct if I assume this will mark it as completed using `IValue()` which by default is true?